pysat/tests/test_utils_io.py

```diff
@@ -208,6 +209,61 @@ def test_write_and_read_netcdf4_w_kwargs(self, wkwargs, lkwargs):
         self.eval_loaded_data()
         return
 
+    @pytest.mark.parametrize("kwargs, target", [
+        ({}, dt.timedelta(seconds=1)),
+        ({'epoch_unit': 'ns'}, dt.timedelta(microseconds=1)),
+        ({'epoch_origin': dt.datetime(1980, 1, 6)}, dt.timedelta(seconds=1)),
+        ({'epoch_unit': 'ns', 'epoch_origin': dt.datetime(1980, 1, 6)},
+         dt.timedelta(microseconds=1))])
+    def test_read_netcdf4_w_epoch_kwargs(self, kwargs, target):
+        """Test success of writing and reading a netCDF4 file.
+
+        Parameters
+        ----------
+        kwargs : dict
+            Optional kwargs to input into `load_netcdf`. Allows the epoch
+            calculation to use custom origin and units.
+        target : dt.timedelta
+            Expected interpretation of 1 sec of time in loaded data when a given
+            epoch unit is specified.  Default unit for pds.to_datetime is 1 ms.
+
+        """
+
+        # TODO(#947) Expand to xarray objects
+        if not self.testInst.pandas_format:
+            pytest.skip('Not yet implemented for xrray')
```

Comment:
```suggestion
            pytest.skip('Not yet implemented for xarray')
```

Reply:
This is a good changee you should implement.

Reply:
Part of #947 (see comment at 232).  The xarray version requires changes in pysatNASA for improved compliance, and I'd rather deal with that downstream.

Reply:
I was referring to the uncommitted spelling change...

Reply:
😅 I was waiting for more comments before triggering another round of github actions